.github/workflows/PR.yml

```diff
@@ -90,21 +90,5 @@ jobs:
           arguments: kotlinUpgradeYarnLock build --stacktrace
           build-root-directory: sample
 
-      - name: Bundle the build report
```

Comment:
Removed because we have build scans now and the junit report action does not have the correct permissions. While we could fix the permissions, with the build scans I don't see a reason.

Reply:
sgtm